src/openms/source/ANALYSIS/ID/SiriusMSConverter.cpp

```diff
@@ -85,7 +86,11 @@ namespace OpenMS
     Peak1D peak;
 
     // monoisotopic_trace
-    peak_index = getHighestIntensityPeakInMZRange(precursor_mz, precursor_spectrum, 10, true);
+    const int tolerance = 10;
```

Comment:
I think it is fine to set the error to 10 ppm (max), since is about finding the peak with the highest intensity based on a test mass-to-charge in the spectrum .e.g the precursor.